docs/features/batch.md

```diff
@@ -487,7 +487,7 @@ classDiagram
 * **`processRecord()`** – If you need to implement asynchronous logic, use this method, otherwise define it in your class with empty logic
 * **`processRecordSync()`** – handles all processing logic for each individual message of a batch, including calling the `recordHandler` (`this.handler`)
 
-You can then use this class as a context manager, or pass it to `processPartialResponseSync` to process the records in your Lambda handler function.
+You can then use this class as a context manager, or pass it to `processPartialResponse` to process the records in your Lambda handler function.
```

Comment:
What do we mean by context manager here? I thought that was a Python thing.

Reply:
Good spot, it's a copy/pasted section from the Python docs. 

Will fix